ext/zlib/zlib.c

```diff
@@ -830,7 +830,7 @@ static bool zlib_create_dictionary_string(HashTable *options, char **dict, size_
 						for (i = 0; i < ZSTR_LEN(*ptr); i++) {
 							if (ZSTR_VAL(*ptr)[i] == 0) {
```

Comment:
You might as well replace this by `zend_str_has_nul_byte` while you are touching the code.

Reply:
Ah, right. Thank you!